src/chrome/content/rules/Wdr.de.xml

```diff
@@ -4,41 +4,46 @@
 		- adaptiv.wdr.de
 		- podcast.wdr.de
 		- print.wdr.de
-		- reportage.wdr.de
 		- www.einslivekrone.de
 		- www.kiraka.wdr.de
 		- www.kirche.wdr.de
+		- www.kirche-im-wdr.de
 		- www.lindenstrasse.de
 		- www.wdrmaus.de
 		- ^wdr.de
 
 	certificatechain broken on:
-		- webmail.wdr.de
 		- freundeskreis.einslive.de
 
 	Non-resolvable host:
 		- static.freundeskreis.einslive.de
 
 	Redirect to plain http on:
-		- digit.wdr.de
 		- www.einsfestival.de
-		- www.planet-wissen.de
+		- wdrmaus.de
 
-	Time-out on;
+	Time-out on:
+		- dom360.wdr.de
 		- klangkiste.wdr.de
-		- mobiltext.wdr.de
+		- lindenstrasse.de
 		- neuneinhalb.wdr.de
-		- ralphsleckerwissen.wdr.de
+		
+	Mixed-Content:
```

Comment:
I only found mixed passive content. Is there anything broken due MCB on that host?